### ISSUE TITLE
a11y - Move focus to drawer menu when opened

### ISSUE THREAD

**Issue by robdodson** [addyosmani/smaller-pictures-app#7]
Opening the drawer menu with a screen reader does not transfer focus to the menu. This means I can continue to move around the page with a screen reader even though the menu is open and the overlay should be covering the page.

Major bonus points if you want to trap the keyboard focus and screenreader inside of the drawer while it's open so a screen reader user won't "escape" the menu. https://accessibility.oit.ncsu.edu/training/aria/modal-window/version-3/
